HeterogeneousCore/SonicTriton/scripts/cmsTriton

```diff
@@ -13,12 +13,14 @@ MODELS=()
 FORCE=""
 CLEANUP=true
 TMPDIR=""
+INSTANCES=0
+LOCALMODELREPO=""
 DRYRUN=""
 PARENTPID=""
 BASEPORT=8000
 AUTOPORT=""
 NPORTS=3
-IMAGE=fastml/triton-torchgeo:22.07-py3-geometric
+IMAGE=fastml/triton-torchgeo:21.06-py3-geometric
```

Comment:
please fix the rebase, this is a regression.

Reply:
Fixed